pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2491,14 +2491,16 @@
       "StableVersions": [
         "4.5.0",
         "4.5.1",
-        "4.5.2"
+        "4.5.2",
+        "4.5.3"
```

Comment:
Has 4.5.3 been built stable yet?

Reply:
No, but I incremented it following the servicing guideline here: https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/library-servicing.md#determine-packageversion

Reply:
I see.  Maybe that document should explicitly call out that `StableVersion` should not be changed until *after* you've shipped the stable release.  `StableVersion` list represents history.

Reply:
@ahsonkhan this change still needs to be removed before I can merge this

Reply:
Done, thanks.